src/pages/experiments/[experimentId]/plots-and-tables/components/AxesDesign.jsx

```diff
@@ -6,6 +7,7 @@ import {
 
 const AxesDesign = (props) => {
   const { onUpdate, config } = props;
+  const onUpdateThrottled = _.debounce((obj) => onUpdate(obj), 10);
```

Comment:
(a) this is meant to be `_.throttle` and not `_.debounce`
(b) you need to wrap it in a `useRef`, like you saw it with the other code. otherwise the debounce function will be recreated every time the component re-renders, so it becomes useless